Adding flat file CSV documentation to the specification doc

nomadaisy opened the pull request:
I added example comma-delimited .txt files to all sections with example XML files. I also added a paragraph describing the flat files and pointing to the csv-templates repository for more information.

jungshadow commented on index.html:
```diff
@@ -74,6 +74,9 @@ <h1 class="title"><abbr title="Voting Information Project">VIP</abbr> Format Spe
           <p>The file containing the VIP feed should be named vipFeed-[<a href="http://www.itl.nist.gov/fipspubs/co-codes/states.htm">FIPS code</a> for state or county]-[election year]-[election month]-[election day].xml. If the file is zipped, the file extensions of .zip or .xml.zip are also acceptable. <br/><br/>For instance, vipFeed-19-2012-11-06.zip denotes Iowa's (the FIPS code for IA is 19) ...
 
+          <h3>Comma-delimited flat files</h3>
+          <p>If your office is producing flat files instead of an XML file, the files must be comma-delimited .txt files. While the format of the files differs slightly from the XML, the attributes included must conform to the field specifications below.<br/><br/> Examples of the flat files are below each XML example, along with the files included to link those elements when the files are compile ...
```
The two `<br />` tags should be `</p><p>` since they divide two separate paragraphs.

jungshadow commented:
This looks good, @nomadaisy. My only comment is that you'll want to explicitly mention the file name in the examples (e.g. for locality, mention above the example that the file name is locality.txt). Since the CSV format relies on separate, distinctly-named files, we should surface that point in these docs (NB: I noticed you have a nice couple of paragraphs at the top, but it should be explicit in the examples as well). It may also help clarify the relationship file examples under `polling_location`, `early_vote_site`, and similar.

nomadaisy commented:
@jungshadow I've added detail about the files next to each example and updated the certification documentation. This should be ready to merge, if you think the flat file documentation is comprehensive enough.

jungshadow commented on examples/contest_result.xml:
```diff
@@ -3,4 +3,5 @@
   <jurisdiction_id>20013</jurisdiction_id>
   <entire_district>no</entire_district>
   <total_votes>100</total_votes>
+  <certification>unofficial_complete</certification>
```
`certification` isn't an element. It's just an attribute. You can just remove this here since the attribute exists (i.e. `certification="certified"`).

jungshadow commented on examples/ballot_line_result.xml:
```diff
@@ -4,4 +4,5 @@
   <candidate_id>22003</candidate_id>
   <entire_district>no</entire_district>
   <votes>24</votes>
+  <certification>certified</certification>
```
`certification` isn't an element. It's just an attribute. You can just remove this here.

jungshadow commented:
So close. The `ballot_line_result` element is missing the `certification="certified"` attribute and both `ballot_line_result` and `contest_result` need the `certification` element removed. Once those changes are in, you're good to merge.

@pstenbjorn After @nomadaisy merges, you may want to merge these changes into the vip5 branch, so we can start the new docs there.

pstenbjorn commented:
@jungshadow happy to do it.  I will be doing some cleanup work over the weekend and on Monday.

jungshadow commented:
Also, @nomadaisy, can you also merge/rebase master into gh-pages to push this live?

nomadaisy commented:
@jungshadow @pstenbjorn Merged and pushed to gh-pages

pstenbjorn commented:
@nomadaisy thank you!
